images/tier2/mobile-brand-admin.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" viewBox="0 0 24 24">
```

Comment:
The whole mobile brand administration tool is going away with Daylight, so I'd be very surprised if we need this anywhere....